packages/api/src/feature/controller.ts

```diff
@@ -0,0 +1,27 @@
+import { Router } from "express";
+import type { Request, Response, NextFunction } from "express";
+
+import { isValidationError } from "../validators/validator_util";
+
+import { featureService } from "./service";
+import { extractIp, verifyUserAuthentication } from "../middleware";
+
+export const featureController = Router();
+
+featureController.post(
+  "/features",
+  verifyUserAuthentication,
+  extractIp,
+  async (_req: Request, res: Response, next: NextFunction): Promise<void> => {
+    try {
+      const featureFlags = await featureService.getFeatureFlags();
+      res.json(featureFlags);
+    } catch (err) {
+      if (isValidationError(err)) {
+        res.status(400).json({ error: err });
+        return;
```

Comment:
Since this endpoint doesn't make use of any inputs, there is nothing to validate, so we needn't check for validation errors